### PR TITLE
defaults:  add page

### PR DESCRIPTION
**Pull request by sfarzy**
OS X command to read or modify user defaults.

### CODE REVIEW

File: pages/osx/defaults.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+`defaults read -app {{app-name}} {{key}}`
+
+- write key value
+
+`defaults write -app {{app-name}} {{key}} {{-type}} {{value}}`
+
+- delete all defaults
+
+`defaults write -app {{app-name}}`
```
Comment (ostera) [inline pages/osx/defaults.md]: was this supposed to be `defaults delete -app {{app-name}}` ?

Comment (sfarzy) [inline pages/osx/defaults.md]: Yes, Missed that error. Thanks for pointing out.
Corrected now and update has been committed.

**Comment by waldyrious** [comment]
Thanks, looks great already :) Just two minor nitpicks:
1. It would probably be better to write {{app_name}} as that's the format we're trying to standardize on in tldr, for separating words in tokens (unless the app names in OS X for this command indeed are specified in lowercase, hyphen-separated form).
2. How about adding a small note to the command that uses `{{-type}}` to explain what it is, e.g. by providing one or two example types?

File: pages/osx/defaults.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# defaults
+
+> Access OS X user defaults
+
+- read system default value
+
+`defaults read {{domain}} {{key}}`
+
+- read default values of applications
```
Comment (igorshubovych) [inline pages/osx/defaults.md]: should it be application, not applicationS, considering the fact that you are talking about some particular {{app_name}}?

Comment (waldyrious) [inline pages/osx/defaults.md]: I think this is just a language subtlety. The description is written in a generic sense, not literally referring to a concrete set of multiple applications. That said, it certainly doesn't hurt to be explicit and change the phrasing to refer to a single app, which is what the command actually does.

File: pages/osx/defaults.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+`defaults read -app {{app_name}} {{key}}`
+
+- write key value
+
+`defaults write {{domain}} {{key}} {{-type}} {{value}}`
+
+- Speed up Mission Control animations
```
Comment (waldyrious) [inline pages/osx/defaults.md]: This is a very specific example which I feel doesn't fit with the purpose of tldr and with the rest of the examples on this page. I'd suggest dropping it.

**Comment by igorshubovych** [comment]
@sfarzy do you need help with completing this?

**Comment by sfarzy** [comment]
@igorshubovych Help is very much appreciated.
I was looking for a simple example.

**Comment by igorshubovych** [comment]
@sfarzy here you can find a several minutes video about how to squash commits:
https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#submitting-a-pull-request (number 9)

File: pages/osx/defaults.md (hunk 3 above)
Comment (igorshubovych) [inline pages/osx/defaults.md]: @waldyrious do we still need to remove it. i feel this example might be ok. in this case, maybe we just can merge it.

**Pull request merged**